meilisearch-http/src/analytics/segment_analytics.rs

```diff
@@ -210,6 +210,21 @@ impl Segment {
                     "server_provider": std::env::var("MEILI_SERVER_PROVIDER").ok(),
             })
         });
+        let info = {
+            let db_path = opt.db_path != PathBuf::from("./data.ms");
+            let import_dump = opt.import_dump.is_some();
+            let dumps_dir = opt.dumps_dir != PathBuf::from("dumps/");
+            let import_snapshot = opt.import_snapshot.is_some();
+            let http_addr = opt.http_addr != "127.0.0.1:7700";
```

Comment:
We should maybe also check that it is different from `localhost:7700` as this is equivalent even if it is not the default value.

Reply:
I can but I think the idea was just to see if the field has been used, not really how people use it :thinking: 
What do you think @gmourier?

Reply:
Absolutely, it's all about whether this option is used.